src/object_detector/filters.jl

```diff
@@ -0,0 +1,50 @@
+
+EDGE_HORIZONTAL = [1 2 1;0 0 0;-1 -2 -1]
```

Comment:
You should change it to `const EDGE_HORIZONTAL = [1 2 1;0 0 0;-1 -2 -1]` for performance and for safety. 

Global constants are okey. Global variable are not good.